### PR TITLE
Fix dependency conflicts

### PR DESCRIPTION
**Pull request by regadas**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -123,8 +123,9 @@ lazy val core: Project = Project(
   moduleName := "featran-core",
   description := "Feature Transformers",
   libraryDependencies ++= Seq(
-    "com.twitter" %% "algebird-core" % algebirdVersion,
+    "com.twitter" %% "algebird-core" % algebirdVersion exclude ("org.typelevel", "algebra_2.11"),
```
Comment (nevillelyh) [inline build.sbt]: Which version are they depending on?

Comment (regadas) [reply build.sbt]: they are depending on `0.7.0` do you think is better to submit a PR on their side?

Comment (nevillelyh) [reply build.sbt]: We can do both, and revert our workaround once they fix it.

Comment (regadas) [reply build.sbt]: the main conflicts here is with `cats` and `shapeless`

Comment (nevillelyh) [reply build.sbt]: I think you can remove the `exclude ()` part, plus it only affects 2.11 while we also cross build for 2.12.

The explicit dependency down there should do it.

Comment (regadas) [reply build.sbt]: the exclude is actually required here ... good point on the cross build. Updated it to allow cross builds.

**Comment by nevillelyh** [comment]
Format error now :)

**Pull request merged**